Code/Editor/Core/QtEditorApplication.cpp

```diff
@@ -44,7 +44,7 @@ enum
 {
     // in milliseconds
     GameModeIdleFrequency = 0,
-    EditorModeIdleFrequency = 1,
+    EditorModeIdleFrequency = 0,
```

Comment:
question: Do we definitely want to change this value or was this something changed during testing?

Reply:
100% intentional, it's important that we tick more often in the Editor now so that we can render in time.

Reply:
Okay good to know, thanks for confirming